### ISSUE TITLE
Financial Intelligence, May 19: Part Two - 5 & 6 

### ISSUE THREAD

**Issue by kunzmann** [Blackmill/book-club#186]
Book: Financial Intelligence by Karen Berman and Joe Knight with John Case
Link: https://financialintelligencebook.com/

Aiming to read:
Part Two: Section 5: Profit is an Estimate
Part Two: Section 6: Cracking the Code of the Income Statement

MC: @lachlanhardy 
Notes: @kunzmann

See you all at 12pm AEST, May 19th @ https://blackmill.co/meet

As always, if you'd like a calendar invite and/or access to Slack beforehand, get in touch via [gday@blackmill.co](mailto:gday@blackmill.co).

**Comment by kunzmann** [comment]
## 5 Profit Is an Estimate & 6 Cracking the Code of the Income Statement

Some of the basic definitions were helpful in connecting dots (nothing profound, but still useful):
'The top line' - means the top line of an income statement
Pro forma - projected data
'The accounting package does it for me'
Highlights the absolute separation between income statement and the day to day. Spreading out the accounting across a time period.
Answering the question 'how much money do we have available?' can be tricky.
Volume can affect what accounting methods are feasible.

Cash accounting vs accrual - ATO forces accrual at a certain revenue level

"Can we afford to pay for this big expenditure right now?" - forecasting & risk analysis are sometimes needed to answer this kind of question

Layoffs - how much of this is risk analysis gone wrong? Severance can be an immediate hit to cash - how do you decide if it is worth it?

Tricky to understand the nuances between day to day cash position (reality) vs estimates, projections, assumptions, judgement calls

Vendors also want these sorts of projections and estimates to make decisions about offering credit & how much.
Then vendors have insurance based on these lines of credit & that affects how much credit they are willing to extend for how long.

High volume / low margin - profit can be tiny, e.g. 20k profit / million units sold

Income statements can sometimes contain obligations that have no exact due date
Printing 10k cards for $150 discounts with no expiry date - in the liability column forever!

What will the market pay - how can you figure out pricing? Underpricing can have long term negative consequences, but maybe it is worth it in the short term for necessary cash flow.

Hiring can be affected sometimes - e.g. no significant changes allowed to an income statement when a company is being sold, sudden hiring freeze

Write ups & write downs
https://www.arnnet.com.au/article/707221/xero-net-profit-dives-113-5m-further-into-red/
They bought a company that was going really poorly.
What is an impairment?
Basically the value they had on the balance sheet was bigger than it actually was. When you buy a company, it is put in as a value you think it is worth.

Many numbers can be looking good - revenue up x%, user growth etc. - but still profit can tank
